node/src/Pos/Wallet/Web/ClientTypes/Instances.hs

```diff
@@ -120,6 +108,19 @@ instance Buildable CAccountInit where
         caInitWId
         caInitMeta
 
+instance Buildable NewBatchPayment where
+    build NewBatchPayment{..} =
+        bprint ("{ from="%build
+                %" to="%(later mapBuilder)
+                %" inputSelectionPolicy="%build
+                %" }")
+        npbFrom
+        npbTo
+        npbInputSelectionPolicy
+
+instance Buildable (SecureLog NewBatchPayment) where
+    build = "<new batch payment>"
```

Comment:
But we still very want to see `inputSelectionPolicy` :slightly_smiling_face:

Reply:
@Martoon-00 would something like this leak info:
```
instance Buildable (SecureLog NewBatchPayment) where
    build (SecureLog NewBatchPayment {..}) =
        -- TODO: there is some duplication from Buildable NewBatchPayment
        -- reuse the code above!
        bprint ("{ from="%build
                %" to="%(later mapBuilder)
                %" inputSelectionPolicy="%build
                %" }")
        (SecureLog npbFrom)
        (map (first SecureLog) npbTo)
        npbInputSelectionPolicy
```

with this we show coin amounts sent, but we don't know sender or recipient address ?

Reply:
Yeah, excellent!

Reply:
@Martoon-00 Fixed here https://github.com/input-output-hk/cardano-sl/pull/1919/commits/bdb95d4edb1e8651705a882abf668ff57d760178 .

The commit above contains some changes already contained in PR https://github.com/input-output-hk/cardano-sl/pull/1858 (not yet merged)

In private discussion with @Martoon-00 we came to conclusion that https://github.com/input-output-hk/cardano-sl/pull/1919#discussion_r151989532 isn't correct as it leaks `Coin` amount. Amount should be hidden in the commit pasted above.